packages/css/src/components/blockquote/blockquote.scss

```diff
@@ -6,7 +6,7 @@
 @import "../../common/hyphenation";
 @import "../../common/text-rendering";
 
-@mixin reset {
```

Comment:
Is this the convention we want? The ticket isn't very clear on that. The convention we had (undocumented) was: call the mixin `reset` if there's only one, and `reset-name-of-thing` when there are multiple resets.

Maybe we should define and document the convention first, before implementing it?

Reply:
The ticket is concice, but indicates actual HTML element names: “Dus `reset-button`, `reset-ul`, `reset-dd` etc.”. So `reset-blockquote` matches that, while some others in this PR do not.